packages/era/src/era.ts

```diff
@@ -111,12 +102,14 @@ export const readBeaconBlock = async (eraData: Uint8Array, offset: number) => {
   }
 
   const slot = indices.blockSlotIndex!.startSlot + offset
-  if (slot < ForkSlots.Altair) return Phase0SignedBeaconBlock.decode(data.data as Uint8Array)
-  else if (slot < ForkSlots.Bellatrix)
-    return AltairSignedBeaconBlock.decode(data.data as Uint8Array)
-  else if (slot < ForkSlots.Capella)
-    return BellatrixSignedBeaconBlock.decode(data.data as Uint8Array)
-  else if (slot < ForkSlots.Deneb) return CapellaSignedBeaconBlock.decode(data.data as Uint8Array)
+  if (slot < ssz.ForkSlots.Altair)
+    return ssz.Phase0SignedBeaconBlock.decode(data.data as Uint8Array)
+  else if (slot < ssz.ForkSlots.Bellatrix)
+    return ssz.AltairSignedBeaconBlock.decode(data.data as Uint8Array)
+  else if (slot < ssz.ForkSlots.Capella)
+    return ssz.BellatrixSignedBeaconBlock.decode(data.data as Uint8Array)
+  else if (slot < ssz.ForkSlots.Deneb)
+    return ssz.CapellaSignedBeaconBlock.decode(data.data as Uint8Array)
   else return ssz.ETH2_TYPES.SignedBeaconBlock.decode(data.data as Uint8Array)
```

Comment:
so the eth2_types has latest i.e. deneb in this case?
hopefully microsigner can provider the better api to get types for some slot

Reply:
Yes.  Per my other comment, will work with Paul on a better structure for these